dcs-dtc/Models/AH64/AH64Configuration.cs

```diff
@@ -99,10 +99,10 @@ internal static AH64Configuration FromCombatFliteXML(AH64Configuration previous,
 
                 var lat = pos.Element("Latitude")?.Value;
                 var lon = pos.Element("Longitude")?.Value;
-                var dLat = double.Parse(lat.Replace('.', ','));
-                var dLon = double.Parse(lon.Replace('.', ','));
+                var dLat = double.Parse(lat.Replace(',', '.'));

```

Comment:
This change doesn't work on systems that use decimal point.
As the is no sure way to check which format is used in the combatflite XML i know of, can be checked as follows:

                double dLat, dLon;
                float elevation;
                if (double.Parse(lon.Replace(',', '.')) > 361)
                {
                    dLat = double.Parse(lat.Replace('.', ','));
                    dLon = double.Parse(lon.Replace('.', ','));
                    float.TryParse(pos.Element("Altitude")?.Value.Replace('.', ','), out elevation);
                }
                else
                {
                    dLat = double.Parse(lat.Replace(',', '.'));
                    dLon = double.Parse(lon.Replace(',', '.'));
                    float.TryParse(pos.Element("Altitude")?.Value.Replace(',', '.'), out elevation);
                }

Reply:
Better coding is probably possible, just can't figure it out atm

Reply:
Unfortunately this is significantly outside my abilities and knowledge - I'll happily admit I was surprised I could even figure out why it wouldn't import on my machine

Reply:
OK, i'll pull it in with your other adjustment & make the alteration.